internal/tools/go.mod

```diff
@@ -5,27 +5,30 @@ go 1.18
 require (
 	github.com/frapposelli/wwhrd v0.4.0
 	github.com/go-enry/go-license-detector/v4 v4.3.0
-	github.com/golangci/golangci-lint v1.47.2
+	github.com/golangci/golangci-lint v1.47.3
 	github.com/goware/modvendor v0.5.0
-	github.com/mgechev/revive v1.2.1
+	github.com/mgechev/revive v1.2.3
 	github.com/stormcat24/protodep v0.0.0-20210106175019-c95189bc461b
 	github.com/vektra/mockery/v2 v2.12.3
 	golang.org/x/mobile v0.0.0-20201217150744-e6ae53a27f4f
 	golang.org/x/perf v0.0.0-20210220033136-40a54f11e909
 	gotest.tools/gotestsum v1.8.2
 )
 
+// we want to force this version, since 1.0.6 is broken on go1.18
+replace github.com/sylvia7788/contextcheck => github.com/sylvia7788/contextcheck v1.0.4
```

Comment:
Is there an issue or link that explains how this is broken and we can link here?

Reply:
The issue I'm facing is that go tries very hard to make me install v1.0.6, even though golangci declares 1.0.4 in its go.mod. And it doesn't compile on go1.18. I did not find an issue with the same problem. Could you try to reproduce (basically remove the replace, `inv -e tidy-all; inv -e install-tools`. Thanks !

Reply:
I reproduced it locally. AIUI, this is not related to Go 1.18; the issue is that contextcheck made a breaking change in v1.0.6 in their public API (specifically, in the signature of `contextcheck.NewAnalyzer`, which now needs to be passed a config struct), and golangci-lint v1.47.3 still uses the old signature. This was fixed on golangci-lint's side on v1.49.0 (https://github.com/golangci/golangci-lint/commit/cc2d97f31dea62c47c55d5b7a93637e242df7ad1).

We need to either update the comment saying that this needs to be removed on v1.49.0 or just bump to v1.49.0.

Reply:
Oh great, let me try bumping to 1.49.0

Reply:
I'm getting a `note: module requires Go 1.19` when trying 1.49.0. I will add a comment

Reply:
😞 that's a pity, well, let's just add a note then

Reply:
done in the last commit